src/System.Net.Http/tests/FunctionalTests/PostScenarioUWPTest.cs

```diff
@@ -0,0 +1,163 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+
```

Comment:
Nit: I saw a lot of people removing this line. I don't know which one we want but we should settle on a single style.